app/Helpers/database/user.php

```diff
@@ -284,9 +284,8 @@ function getUserListByPerms(int $sortBy, int $offset, int $count, ?array &$dataO
         default => "ua.User ASC ",
     };
 
-    // TODO slow query (70) when ordering by NumAwarded
     $query = "SELECT ua.ID, ua.User, ua.RAPoints, ua.TrueRAPoints, ua.LastLogin,
-                (SELECT COUNT(*) AS NumAwarded FROM Awarded AS aw WHERE aw.User = ua.User) NumAwarded
+                ua.achievements_unlocked NumAwarded
```

Comment:
This also fixes an issue where hardcore unlocks were counted twice because the subquery doesn't have a filter on `HardcoreMode` or a `group by AchievementID` clause.

Before:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/592ece69-b982-4925-9858-efff8650d6bf)

After:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/02725f01-95c3-4831-bd5b-9500d4e54af0)

Profile:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/3e9e1380-9ef1-4b13-a59b-126e9d8cb8e1)

12049+90=12139

Reply:
wonderful! i wondered why there were no additional filters 😅